From c4fbcd54d3e58076ba6ec79f4af82744de54aecc Mon Sep 17 00:00:00 2001
From: Christophe Giboudeaux <christophe@krop.fr>
Date: Mon, 25 Mar 2019 17:29:44 +0100
Subject: [PATCH] Prepare the kdelibs4 removal.

kdelibs is only used to get installation path. we'll hardcode them instead.
---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 730ed91..be386d8 100644
--- a/Makefile
+++ b/Makefile
@@ -165,8 +165,8 @@ endif
 ICON_PREFIX:=$(DESTDIR)$(PREFIX)/share/pixmaps
 X11_ICON_PREFIX:=$(DESTDIR)$(PREFIX)/X11R6/include/X11/pixmaps
 GNOME_PREFIX:=$(shell gnome-config --prefix 2> /dev/null)
-KDE_PREFIX:=$(shell kde-config --install apps --expandvars 2> /dev/null)
-KDE_ICON_PREFIX:=$(shell kde-config --install icon --expandvars 2> /dev/null)
+KDE_PREFIX:=$(DESTDIR)$(PREFIX)/share/applications
+KDE_ICON_PREFIX:=$(DESTDIR)$(PREFIX)/share/icons
 
 # Maemo flag
 MAEMOFLAG:=
-- 
2.21.0

